src/panels/lovelace/special-rows/hui-select-row.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  customElement,
+  property,
+  css,
+  CSSResult,
+} from "lit-element";
+
+import "@polymer/paper-checkbox/paper-checkbox";
```

Comment:
Can we use https://material-components.github.io/material-components-web-components/demos/checkbox.html ?

Reply:
It does work, I just can't find the variable to style the color?! Would expect `--mdc-theme-primary` to do that?